test/service/test_response.py

```diff
@@ -1451,6 +1468,81 @@ def test_disease_facet(self):
+        """
+        url = self.base_url + "/index/projects"
+        test_data = {
+            # project has 1 donor organism
+            '627cb0ba-b8a1-405a-b58f-0add82c3d635': {
+                'organismAge': [{'term': {'value': '20', 'unit': 'year'}, 'count': 1}],
+                'organismAgeUnit': [{'term': 'year', 'count': 1}],
+                'organismAgeValue': [{'term': '20', 'count': 1}],
+
+            },
+            # project has multiple donor organisms
+            '2c4724a4-7252-409e-b008-ff5c127c7e89': {
+                'organismAge': [{'term': {'value': '40-44', 'unit': 'year'}, 'count': 1},
+                                {'term': {'value': '55-59', 'unit': 'year'}, 'count': 1}],
+                'organismAgeUnit': [{'term': 'year', 'count': 1}],
+                'organismAgeValue': [{'term': '40-44', 'count': 1}, {'term': '55-59', 'count': 1}]
+            },
+            # project has 1 donor organism but no organism_age properties
+            'c765e3f9-7cfc-4501-8832-79e5f7abd321': {
+                'organismAge': [{'term': {'value': None, 'unit': None}, 'count': 1}],
+                'organismAgeUnit': [{'term': None, 'count': 1}],
+                'organismAgeValue': [{'term': None, 'count': 1}],
+            }
+        }
+        self._verify_facet_content(test_data, url)
+
+    def test_organism_age_facet_search(self):
+        """
+        Verify the values of filtered search on organism age facet
+        """
+        test_data = {
+            'entryId': '627cb0ba-b8a1-405a-b58f-0add82c3d635',
+            'donorOrganisms': [
+                {
+                    'id': [
+                        'donor_ID_1'
+                    ],
+                    'donorCount': 1,
+                    'genusSpecies': [
+                        'Homo sapiens'
+                    ],
+                    'organismAge': [
+                        '20'
+                    ],
+                    'organismAgeUnit': [
+                        'year'
+                    ],
+                    'organismAgeRange': [
+                        {
+                            'gte': 630720000.0,
+                            'lte': 630720000.0
+                        }
+                    ],
+                    'biologicalSex': [
+                        'male'
+                    ],
+                    'disease': [
+                        'H syndrome'
+                    ]
+                }
+            ]
+        }
+        url = self.base_url + "/index/projects"
+        filters = {'organismAge': {'is': [{'value': '20', 'unit': 'year'}]}}
+        params = {'filters': json.dumps(filters)}
+        response = requests.get(url, params=params)
+        response.raise_for_status()
+        response_json = response.json()
+        hit = one(response_json['hits'])
+        for entity_field, expected_data in test_data.items():
+            self.assertEqual(hit[entity_field], expected_data)
```

Comment:
```suggestion
        self.assertEqual(hit['entryId'], '627cb0ba-b8a1-405a-b58f-0add82c3d635')
        self.assertEqual(one(hit['donorOrganisms'])['organismAge'], ['20'])
        self.assertEqual(one(hit['donorOrganisms'])['organismAgeUnit'], ['year'])
```
These 3 properties should be enough to confirm the expected hit was returned without the need for all the other values in `test_data`